src/redis-benchmark.c

```diff
@@ -971,6 +1040,40 @@ static void startBenchmarkThreads() {
         pthread_join(config.threads[i]->thread, NULL);
 }
 
+static void initQpsControl() {
+    int num_threads = config.num_threads == 0 ? 1 : config.num_threads;
+    int qps_per_thread = config.qps / num_threads;
+    config.paused_clients = zmalloc(num_threads * sizeof(list *));
+    config.last_resume_time = zmalloc(num_threads * sizeof(long long));
+    config.count = zmalloc(num_threads * sizeof(int));
```

Comment:
We should make sure the config.count and config .pause are initialized
either use zcalloc or in the following for loop

Reply:
fixed